clair/clair.go

```diff
@@ -0,0 +1,209 @@
+package clair
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/containerops/wrench/setting"
+	"github.com/coreos/clair/database"
+	"github.com/coreos/clair/updater"
+	"github.com/coreos/clair/utils"
+	"github.com/coreos/clair/utils/types"
+	"github.com/coreos/clair/worker"
+	"github.com/coreos/pkg/capnslog"
+
+	_ "github.com/coreos/clair/updater/fetchers"
+	_ "github.com/coreos/clair/worker/detectors/os"
+	_ "github.com/coreos/clair/worker/detectors/packages"
+)
```

Comment:
package import should follow the format as below,please update.
import (
    go packages

    the third packages

    dockyard packages
)